readthedocs/core/views/__init__.py

```diff
@@ -48,17 +48,25 @@ def get_context_data(self, **kwargs):
         return context
 
 
-def server_error_404(request, template_name="404.html", exception=None):
+def server_error_404(
+    request, template_name="errors/404/community.html", exception=None
```

Comment:
I didn't get far with this thought, but I had the idea that this would be configured in Read the Docs for Business through it's URL conf. I'll get back to it :+1:

Reply:
It's gone :+1: I don't think we need to work with any alternative templates or settings here.